packages/apollo-server-hapi/src/ApolloServer.ts

```diff
@@ -33,10 +32,9 @@ export interface HapiListenOptions {
   launcherOptions?: EngineLauncherOptions;
 }
 
-const handleFileUploads = (
-  uploadsConfig: Record<string, any>,
-  server: ApolloServerBase<hapi.Request>,
-) => async (req: hapi.Request, h: hapi.ResponseToolkit) => {
+const handleFileUploads = (uploadsConfig: Record<string, any>) => async (
```

Comment:
These changes are large enough I want to make sure it's not a bug that server wasn't used

Reply:
server used to be used when the configuration was in the server constructor. Now that it's configured in the `registerServer` call, we con't have to worry about it